transformers/component_tests/queries/get_urns_by_ilk_query_test.go

```diff
@@ -248,20 +262,40 @@ var _ = Describe("All Urns function", func() {
 			helper.AssertUrn(actualUrn, expectedUrn)
 		})
 
-		It("gets urn state with updated values", func() {
+		It("gets latest urn state with updated values as of block three", func() {
+			createdTimestamp := helper.GetExpectedTimestamp(timestampOne)
+			updatedTimestampThree := helper.GetExpectedTimestamp(timestampThree)
+			expectedUrn := helper.UrnState{
+				UrnIdentifier: urnOne,
+				IlkIdentifier: helper.FakeIlk.Identifier,
+				BlockHeight:   blockThree,
+				Ink:           strconv.Itoa(blockThreeInk),
+				Art:           strconv.Itoa(setupDataOne[vat.UrnArt].(int)), // Not changed
+				Created:       helper.GetValidNullString(createdTimestamp),
+				Updated:       helper.GetValidNullString(updatedTimestampThree),
+			}
+
+			err = db.Get(&actualUrn, urnsByIlkQuery, helper.FakeIlk.Identifier, blockThree)
+			Expect(err).NotTo(HaveOccurred())
+
+			helper.AssertUrn(actualUrn, expectedUrn)
+		})
+
+		It("gets urn state as of block two by filtering by both min and max block height", func() {
 			createdTimestamp := helper.GetExpectedTimestamp(timestampOne)
 			updatedTimestampTwo := helper.GetExpectedTimestamp(timestampTwo)
 			expectedUrn := helper.UrnState{
 				UrnIdentifier: urnOne,
 				IlkIdentifier: helper.FakeIlk.Identifier,
 				BlockHeight:   blockTwo,
-				Ink:           strconv.Itoa(updatedInk),
+				Ink:           strconv.Itoa(blockTwoInk),
 				Art:           strconv.Itoa(setupDataOne[vat.UrnArt].(int)), // Not changed
 				Created:       helper.GetValidNullString(createdTimestamp),
 				Updated:       helper.GetValidNullString(updatedTimestampTwo),
 			}
 
-			err = db.Get(&actualUrn, urnsByIlkQuery, helper.FakeIlk.Identifier, blockTwo)
+			urnsByIlkMinHeightQuery := `SELECT urn_identifier, ilk_identifier, block_height, ink, art, created, updated FROM api.get_urns_by_ilk($1, $2, $3)`
```

Comment:
I'm thinking it'd be cool to write this test such that we were passing a min but no max, since that's the use case behind this PR - right? But I also get that it may be difficult to do that because of argument order, which has me wondering - how does postgraphile handle a case where we only pass a min (but no max)? Maybe worth firing up postgraphile against a local DB with these changes to verify we can actually execute that call, if you haven't already

Reply:
Good call. I'll explicitly verify this, but since there are defaults for all args, I believe Postgraphile inherits the default (`api.max_block()`) when you only pass a min.

Reply:
If so, then this LGTM 👍